collections/concepts/understanding-permissions.md

```diff
@@ -0,0 +1,120 @@
+  "type": "social_security_number",
+  "data": "XXX-XX-6789",
+  "privacy": {
+    "classification": "pii",
+    "impact_level": "high",
+    "restriction_policy": "mask"
+  },
+  "tenant_id": "c28392d1-15ec-479a-ac15-bbe2764ad66d",
+  "created_by": "b848d641-71fb-439d-808a-efbb8237c282",
+  "created_at": "2022-04-20T23:12:10.401Z"
+}
+```
+
+## Permissions
+
+Each [Application Type](https://docs.basistheory.com/#applications-application-types) has its own set of permissions to choose from (view the [full list here](https://docs.basistheory.com/#permissions-permission-types)).  `Client-side` and `Elements` Applications are only able to `create` Tokens, ensuring that you are able to safely use the Application Key in your client-side code without worrying about a malicious actor using that key to decrypt your data. `Management` Applications can manage infrastructure items only, such as Tenants and Applications, and they cannot interact with Tokens; all other Application Types are restricted to data operations with Tokens. Token permissions come in the format `token:<classification>:<operation>:<scope?>`, for instance `token:bank:read:low` or `token:pci:use:proxy`.
+
+Token Permissions control three things:
+
+1. What Classification of Tokens the Application has access to interact with
+2. What operations the Application can use (create, delete, use, etc.)
+3. How the detokenized plain text data is returned when reading a Token
+
+The `<classification>` segment of the permission tells you what Classification of Token the Application can interact with. If an Application is only granted the `token:pii:create` permission, it cannot create a Token with the Classification `pci` or `bank`.
+
+Which operations an Application can use is likewise controlled by the permissions. If an Application has the `token:pii:delete` permission only, it cannot create any Tokens, only delete Tokens with the Classification of `pii`. `use` is an additional operation to the standard CRUD operations that allows an Application to detokenize Tokens of that classification within the [Proxy](https://developers.basistheory.com/concepts/what-is-the-proxy/) or [Reactors](https://developers.basistheory.com/concepts/what-are-reactors/) (`*:use:proxy` or `*:use:reactor` respectively).
+
+The last consideration when determining what permissions to grant an Application is the Impact Level. Applications can only `read` Tokens that have the same or lower Impact Level than the permission. For instance, if an Application has the `token:pii:read:high` permission, it can read all Tokens classified as `pii` regardless of their Impact Level. However, if the Application only has `token:pii:read:low`, the only Tokens it can fully detokenize are those with the `low` Impact Level. If the Application tries to read a Token with a higher Impact Level (in this example, `moderate` or `high`), the data will be returned either masked or redacted, depending on the Restriction Policy assigned to the Token.
```

Comment:
> Applications can only `read` Tokens that have the same or lower Impact Level than the permission.

This may be ambiguous. Applications can `read` can read any token in the classification, but the restriction policy will be enforced for any Tokens with a high impact level. Maybe we clarify that with: 

```suggestion
The last consideration when determining what permissions to grant an Application is the Impact Level. Applications can only `read` Tokens in plaintext that have the same or lower Impact Level than the permission. For instance, if an Application has the `token:pii:read:high` permission, it can read all Tokens classified as `pii` regardless of their Impact Level. However, if the Application only has `token:pii:read:low`, the only Tokens it can fully detokenize are those with the `low` Impact Level. If the Application tries to read a Token with a higher Impact Level (in this example, `moderate` or `high`), the data will be returned either masked or redacted, depending on the Restriction Policy assigned to the Token.
```

Reply:
done!